features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -942,6 +942,20 @@
         <UseIdentityClaims>true</UseIdentityClaims>
     </AccountSuspension>
 
+    <!--This configuration is for enable/disable reCaptcha feature.-->
+    <SSOLogin>
+        <Recaptcha>
+            <!--Enabling this configuration will prompt reCaptcha after max failed attempts.-->
```

Comment:
Default configuration used to enable reCaptcha in login flow.

Reply:
Addressed with commit 0772ae3e49f675e963852d13acfea631f12dead0